bigchaindb/common/utils.py

```diff
@@ -5,6 +5,8 @@
 import bigchaindb
 from bigchaindb.common.exceptions import ValidationError
 
+VALIDATOR_UPDATE_ID = 'validator01'
```

Comment:
nit: `validator01` is a bit ambiguous in terms of the idea, that this used to block new requests to add/remove validators.

Reply:
@muawiakh I just want a string which I can use as a unique id. If you have any suggestions on better naming do let me know.

Reply:
Maybe change `validator01` to `a_unique_id_string` so it becomes super-clear what its purpose is.